CHANGES.rst

```diff
@@ -1,10 +1,12 @@
 Change History
 ***************
 
-1.3.1 (unreleased)
-==================
+1.4 (unreleased)
+================
+
+- Claim support for Python 3.6 and 3.7.
 
-- Nothing changed yet.
+- Drop Python 2.6 and 3.3.
```

Comment:
Ah, I see that was your intention.

Reply:
Yes, there is always one more place to change.